procrastinate/sql/structure.sql

```diff
@@ -37,7 +37,7 @@ CREATE TABLE procrastinate_events (
 );
 
 CREATE UNLOGGED TABLE procrastinate_job_locks (
-    object text NOT NULL
+    object text NOT NULL UNIQUE
```

Comment:
Other name is PK. The DBA CI doesn't accept this (fixed on the peopledoc/sql-ddl-migration#3251) as it means this table is declared without PK

Reply:
I'm definitely fine with PK :smile: